samples/keycloak-pkce-cmdline-sample/src/main/java/com/google/api/services/samples/keycloak/cmdline/PKCESample.java

```diff
@@ -0,0 +1,74 @@
+package com.google.api.services.samples.keycloak.cmdline;
```

Comment:
All the source files in the repo need a license header

Reply:
done, along with some more documentation.